pandas/io/formats/style.py

```diff
@@ -545,6 +552,7 @@ def render(self, **kwargs) -> str:
         # so we have the nested anys below
         trimmed = [x for x in d["cellstyle"] if any(any(y) for y in x["props"])]
         d["cellstyle"] = trimmed
+        self._render_tooltips(d)
```

Comment:
why wouldn't this happen *after* the update?

Reply:
becuase it is really an extension of the _translate method.